### ISSUE TITLE
Bug when entering the world

### ISSUE THREAD

**Issue by VirusSavion** [Luke100000/minecraft-comes-alive#598]
The error lies in the fact that when entering the world, I am again given a choice of character. And not so long ago my world just got re-created and I lost my survival progress. I don't know how to show and demonstrate it, but I have a permanent bug.

**Comment by Luke100000** [comment]
Are you sure it's mca related? The world reset sounds like outside the scope of mca.

**Comment by VirusSavion** [comment]
> Вы уверены, что это связано с МКА? Сброс мира звучит как выход за рамки mca.

Yes, I'm rather sure that this is due to the mod, since every time I enter the same world I said it makes it possible to choose a character for myself (although I don't want to choose other traits, but leave the one who is).
And according to the theory, there are no other mods of properties on the MCA.

Also I forgot to say the version
Forge Version 36.2.39
Minecraft 1.16.5
MCA Reborn mod version - minecraft-comes-alive-7.3.23+1.16.5-universal
